pocs/tests/utils/test_image_utils.py

```diff
@@ -7,6 +9,22 @@
 from pocs.utils import images as img_utils
 
 
+def test_make_images_dir(save_environ):
+    assert img_utils.make_images_dir()
+
+    # Invalid parent directory for 'images'.
+    os.environ['PANDIR'] = '/dev/null/'
+    with pytest.warns(UserWarning):
+        assert img_utils.make_images_dir() is None
+
+    # Valid parents for 'images' that need to be created.
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
We have a fixture for a [temp images dir](https://github.com/panoptes/POCS/blob/42e717a2099e5714757e0d5e503919d865535d7f/pocs/tests/conftest.py#L20), can that be used? It already appends an `images` to the end but could still test the `make_images_dir` functionality.